src/plugins/blocksmith/components/blocksmith/index.jsx

```diff
@@ -1,1862 +1,10 @@
-                  onShiftBlock={this.handleShiftBlock}
-                  onToggleSpecialCharacterModal={this.handleToggleSpecialCharacterModal}
-                  resetFlag={blockButtonsStyle.top}
-                />
-              </div>
-              <div style={sidebarHolderStyle} className="sidebar-holder">
-                <Sidebar
-                  isHoverInsertMode={isHoverInsertMode}
-                  isOpen={isSidebarOpen}
-                  onToggleSidebar={this.handleToggleSidebar}
-                  onToggleBlockModal={this.handleToggleBlockModal}
-                  onHoverInsert={this.handleHoverInsert}
-                  resetFlag={sidebarResetFlag}
-                  popoverRef={this.popoverRef}
-                />
-              </div>
-              <InlineToolbar>
-                {(props) => (
-                  <>
-                    <BoldButton {...props} />
-                    <ItalicButton {...props} />
-                    <UnderlineButton {...props} />
-                    <LinkButton
-                      {...props}
-                      onToggleLinkModal={this.handleToggleLinkModal}
-                      getEditorState={this.getEditorState}
-                    />
-                    <OrderedListButton {...props} />
-                    <UnorderedListButton {...props} />
-                    <StrikethroughButton {...props} />
-                    <HighlightButton {...props} />
-                  </>
-                )}
-              </InlineToolbar>
-              {Modal && (
-              <ModalErrorBoundary onCloseModal={this.handleCloseModal}>
-                <Modal />
-              </ModalErrorBoundary>
-              )}
-            </div>
-            {!readOnly && (
-              <div className="text-center mt-2">
-                <span className="btn-hover">
-                  <Button id="add-block-button" radius="circle" color="success" size="sm" onClick={this.handleAddEmptyBlockAtEnd}>
-                    <Icon imgSrc="plus" size="md" />
-                  </Button>
-                </span>
-                <UncontrolledTooltip key="tooltip" placement="bottom" target="add-block-button">Add empty block at end</UncontrolledTooltip>
-              </div>
-            )}
-          </ErrorBoundary>
-          {!!Object.keys(errors).length && (
-          <>
-            <p>One or more errors have occurred. Please check your work, save, and report the issue to support.</p>
-            {Object.values(errors).map(({ block, error }) => (
-              <div key={block.getKey()}>
-                <FormText color="danger">{error}</FormText>
-                <FormText color="danger">{block.toString()}</FormText>
-              </div>
-            ))}
-          </>
-          )}
-        </CardBody>
-      </Card>
-    );
-  }
-}
-
-export default connect(selector, createDelegateFactory(delegateFactory))(Blocksmith);
+export default ({ ...rest }) => (
+  <OuterErrorBoundary>
```

Comment:
@hunyoboy can you inline this? odd to have `OuterErrorBoundary` inside the `error-boundary` dir